prototype_source/inductor_with_x86_cpu_as_backend_of_pt2e.rst

```diff
@@ -0,0 +1,182 @@
+TorchInductor with X86 CPU as backend of Quantization in PyTorch 2.0 Export
+=============================================================================
+
+**Author**: `Leslie Fang <https://github.com/leslie-fang-intel>`_, `Weiwen Xia <https://github.com/Xia-Weiwen>`_, `Jiong Gong <https://github.com/jgong5>`_, `Jerry Zhang <https://github.com/jerryzh168>`_
+
+Prerequisites
+^^^^^^^^^^^^^^^
+
+-  `PyTorch 2.0 Export Post Training Static Quantization <https://pytorch.org/tutorials/prototype/pt2e_quant_ptq_static.html>`_
+-  `TorchInductor and torch.compile concepts in PyTorch <https://pytorch.org/tutorials/intermediate/torch_compile_tutorial.html>`_
+
+Introduction
+^^^^^^^^^^^^^^
+
+This tutorial introduces the steps to use PyTorch 2.0 Export Quantization flow and TorchInductor to do post-training static quantization
+in graph mode on X86 CPU devices.
+
+- New Quantization 2.0 flow uses the PT2 Export workflow to capture the model into a graph and perform quantization transformations on top of the ATen dialect graph. This approach is expected to have significantly higher model coverage, better programmability, and a simplified UX.
+- TorchInductor is the new compiler backend that compiles the FX Graphs generated by TorchDynamo into optimized C++/Triton kernels.
+
+The high-level architecture of quantization 2.0 with Inductor could look like this:
```

Comment:
Can we also explain a little bit at the high level about this flow? In particular, the three steps shown in the example below.

Reply:
Thanks for the comment. Added more explanation of this flow at high-level.